### PR TITLE
#24 modified test for --cwd

### PR DESCRIPTION
**Pull request by vyomkeshj**
Made the test more comprehensive

### CODE REVIEW

File: tests/test_job.py [hunk 1]
```diff
@@ -79,19 +79,32 @@ def test_custom_name(hq_env: HqEnv, tmp_path):
...
+    test_file_path = str(tmpdir) + "/test_dir/"
+
+    cwd_submit_tbl = hq_env.command(
         [
             "submit",
-            "sleep 1",
-            "--cwd=/test/directory/",
+            "--cwd=" + test_file_path,
+            "--",
+            "bash",
```
Comment (Kobzol) [inline tests/test_job.py:96]: I suppose that using `[..., "cat", "testfile"]` should also work, but that's a detail :)

**Review by Kobzol — APPROVED** [review]
LGTM.

You'll need to rebase on top of main. Let me know if you want some hints for that :)
```bash
$ git fetch --all
$ git rebase origin/main
(resolve potential conflicts and commit)
$ git push --force
```

File: tests/test_job.py [hunk 2]
```diff
@@ -79,20 +79,32 @@ def test_custom_name(hq_env: HqEnv, tmp_path):
...
-    offset = 10
+def test_custom_working_dir(hq_env: HqEnv, tmpdir):
     hq_env.start_server()
-    cwd_custom_tbl = hq_env.command(
+
+    cwd_offset = 9
+    test_string = "cwd_test_string"
+    test_file = tmpdir.mkdir("test_dir").join("testfile")
+    test_file.write(test_string)
+    test_file_path = str(tmpdir) + "/test_dir/"
```
Comment (spirali) [inline tests/test_job.py:89]: You can just capture path when created without own concatenating of strings:
```python
test_path = tmpdir.mkdir("test_dir")
test_file = test_path.join("testfile")
...
"--cwd=" + str(test_path)
```

**Comment by Kobzol** [comment]
Thanks! :+1:

**Pull request merged**